### PR TITLE
Task contact section

### PR DESCRIPTION
**Pull request by milaferron**
Completed Contact section ready for review. I will need to send my video file via slack.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,8 +30,22 @@
 			<h1> Jamila Ferron</h1>
 			<p> Full-stack web developer with an interest in PHP web development</p>
 		</div>
-		<img class="scroll-icon" src="img/download.svg" alt=""/>
+		<a href="#contact-section"><img class="scroll-icon" src="img/download.svg" alt=""/></a>
 	</header>
+	<section class="container" id="contact-section">
+		<div class="contact-wrapper">
```
Comment (katestarks) [inline index.html]: Is div container needed?

File: index.html [hunk 2]
```diff
@@ -29,8 +30,22 @@
...
+					Reach Out!
+				</h1>
+			</div>
+			<a href="mailto:jamila_ferron@hotmail.co.uk"><img class="contact-icon" src="img/envelope.svg" alt=""/></a>
+			<a href="tel:07534955581"><img class="contact-icon" src="img/phone.svg" alt=""/></a>
+		</div>
+	</section>
 	<footer>
 		<a href="https://github.com/jamilaferron"><img class="footer-icon" src="img/github-sign.svg" alt=""/></a>
 		<a href="https://www.linkedin.com/in/jamila-ferron-6491938a/"><img class="footer-icon" src="img/linkedin.svg" alt=""/></a>
```
Comment (katestarks) [inline index.html:49]: Links to the LinkedIn sign up page - not sure if there is a way around this?

File: css/style.css [hunk 3]
```diff
@@ -105,6 +101,44 @@ header {
     left: 50%;
 }
 
+.contact-wrapper {
+    width: 100%;
```
Comment (katestarks) [inline css/style.css]: Block element so already 100% by default

File: css/style.css [hunk 4]
```diff
@@ -105,6 +101,44 @@ header {
...
+
+.section-header {
+    padding-top: 10px;
+    height: 250px;
+}
+
+.front-text {
+    color: #ffffff;
+    font-family: 'Muli', sans-serif;
+    font-weight: 700;
```
Comment (katestarks) [inline css/style.css]: Not needed?

File: css/style.css [hunk 5]
```diff
@@ -105,6 +101,44 @@ header {
...
+}
+
+.shadow-text {
+    font-family: 'Covered By Your Grace', cursive;
+    color: #A9BCD0;
+    opacity: 0.3;
+    font-size: 7rem;
+    position: relative;
+    top: -173px;
+    z-index: 1;
```
Comment (katestarks) [inline css/style.css:130]: Not needed

File: index.html [hunk 6]
```diff
@@ -29,8 +30,22 @@
...
+		<div class="contact-wrapper">
+			<div class="section-header">
+				<h1 class="front-text">
+					Contact
+				</h1>
+				<h1 class="shadow-text">
+					Reach Out!
+				</h1>
+			</div>
+			<a href="mailto:jamila_ferron@hotmail.co.uk"><img class="contact-icon" src="img/envelope.svg" alt=""/></a>
```
Comment (katestarks) [inline index.html]: Not sure why but the clickable area is quite tall on the envelope. Would be better if it was limited to the icon itself.

File: index.html [hunk 7]
```diff
@@ -29,8 +30,22 @@
...
+			<a href="tel:07534955581"><img class="contact-icon" src="img/phone.svg" alt=""/></a>
```
Comment (katestarks) [inline index.html]: Same as previous

**Review by katestarks — CHANGES_REQUESTED** [review]
Looks good - just a couple of lines of code that you might not need

File: css/style.css (hunk 5 above)
Comment (milaferron) [reply css/style.css:130]: I have included a z-index here as Mike said it was good practice.

**Review by katestarks — APPROVED** [review]

**Pull request merged**